pkg/i18n/i18n.go

```diff
@@ -79,6 +81,17 @@ func (l *Localizer) GetLanguage() string {
 
 // add translation file(s)
 func addBundles(i18nBundle *i18n.Bundle) {
-	addDutch(i18nBundle)
-	addEnglish(i18nBundle)
+	err := addPolish(i18nBundle)
+	if err != nil {
+		log.Fatal(err)
```

Comment:
just one small thing: could you pass the *Logrus.logger log into this function and then call fatal on that?